test/e2e/data/cni/calico/calico.yaml

```diff
@@ -0,0 +1,4724 @@
+---
```

Comment:
Just a bit confused because of our previous discussion. 

This YAML is just the same as the one referenced in our quickstart guide, right? (https://raw.githubusercontent.com/projectcalico/calico/v3.24.1/manifests/calico.yaml)